cli/azd/cmd/infra_create.go

```diff
@@ -72,11 +72,15 @@ func (ica *infraCreateAction) Run(ctx context.Context, cmd *cobra.Command, args
 		return fmt.Errorf("loading environment: %w", err)
 	}
 
-	_, err = project.LoadProjectConfig(azdCtx.ProjectPath(), &environment.Environment{})
+	proj, err := project.LoadProjectConfig(azdCtx.ProjectPath(), &environment.Environment{})
 	if err != nil {
 		return fmt.Errorf("loading project: %w", err)
 	}
 
+	if err = project.Initialize(ctx, proj); err != nil {
```

Comment:
Then this becomes an instance call.
```suggestion
	if err = proj.Initialize(ctx); err != nil {
```